### ISSUE TITLE
There are still some missing Acc.Ids

### ISSUE THREAD

**Issue by isabelrios** [mozilla-lockbox/lockbox-ios#602]
The list is smaller but there is still a few `accessibility identifier`s that are missing:

**Preferred browser** options:

- [ ] Safari
- [ ] Firefox
- [ ] Google Chrome

**Sort Entries button**, not the button itself but the label:

- [ ] Select options for sorting your list of entries (currently Recent)
- [ ] Select options for sorting your list of entries (currently A-Z)

**Sort Entries** action sheet menu:

- [ ] Alphabetically
- [ ] Recently Used

I think those are all, but this is a live list, so please feel free to add more...

**Comment by joeyg** [comment]
I am not sure accessibility identifiers can be added to action sheet items - @sashei any ideas?
